admin-dev/themes/default/webpack.config.js

```diff
@@ -90,6 +92,17 @@ module.exports = (env, argv) => {
       new MiniCssExtractPlugin({
         filename: 'theme.css',
       }),
+      new HtmlWebpackPlugin({
+        filename: 'preload.tpl',
+        templateContent: '{{{preloadLinks}}}',
+        inject: false,
+      }),
+      new FontPreloadPlugin({
+        index: 'preload.tpl',
+        extensions: ['woff2', 'woff', 'ttf', 'eot'],
```

Comment:
@NeOMakinG preload only woff2 format. You are loading every single file on page load.

Reply:
Damn, mb 🤣